simulator/config/config.go

```diff
@@ -85,73 +94,86 @@ func NewConfig() (*Config, error) {
 	}, nil
 }
 
```

Comment:
Could you keep the original way to set configuration via env? We're using this way for a long time and some users may want to use that.